.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  snyk:
+    strategy:
+      matrix:
+        version: [7/community, 8/community, 8/developer, 8/enterprise]
+        include:
+          - version: 7/community
+            tag: 7-community
+          - version: 8/community
+            tag: 8-community
+          - version: 8/developer
+            tag: 8-developer
+          - version: 8/enterprise
+            tag: 8-enterprise
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build -t "sonarqube:${{ matrix.tag }}" "${{ matrix.version }}"
+      - name: Run Snyk
+        uses: snyk/actions/docker@master
+        env:
+          SNYK_TOKEN: ${{secrets.SNYK_TOKEN}}
+        with:
+          command: test
```

Comment:
Could this be conditional on the event? The two workflows could be merged, then (essentially the same?)

Reply:
I refactored it a bit, it's better now but since github actions don't support YAML anchors, there is a bit of duplication anyway